tests/dotnetCampus.CommandLine.Analyzer.Test/dotnetCampusCommandLineAnalyzerUnitTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿//using Microsoft.CodeAnalysis;
```

Comment:
why comment？

Reply:
The unit test packages that the default template references are all disappeared so no test codes can be compiled.